oak_functions/loader/src/server.rs

```diff
@@ -493,13 +493,13 @@ where
         // The `function` did not terminate within the policy timeout
         None => Response::create(
             StatusCode::PolicyTimeViolation,
-            "Reason: response not available".as_bytes().to_vec(),
+            "Reason: response not available.\n".as_bytes().to_vec(),
```

Comment:
Personally I would not add newlines to these strings, or are they needed for something? Adding newlines makes it harder to include them as part of other messages.

Reply:
This is the response body, it is not being included in any other messages. Is it? 
Adding the newline gives more readability when the message is printed in stdout. Although, now we only support gRPC and the response is wrapped in a tonic::Response. So, I think it does not matter.